docs/bridge/prover_service.mdx

```diff
@@ -3,15 +3,33 @@ title: Bridge Service
 sidebar_position: 6
 ---
 
-With the bridge service a user or an application can initiate token transfers through the bridge or more generally execute cross shard calls. 
+The bridge service is a crucial component that facilitates token transfers and enables cross-shard calls between different blockchain networks. This section focuses on the transfer of NEAR tokens from the mainnet to a Calimero shard as an example use case.
```

Comment:
```suggestion
The bridge service is a component that facilitates token transfers and enables cross-shard calls between different blockchains. This section focuses on the transfer of NEAR tokens from the mainnet to a Calimero shard as an example use case.
```
crucial is too much in my opinion, all bridge components are crucial, as the bridge would not work without any of them missing

Reply:
Okay 👌